test/integration_test/common_test.go

```diff
@@ -695,7 +748,7 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace, projectMappings strin
 	}
 
 	// Get cluster pair details for destination cluster
-	destInfo, err := volumeDriver.GetClusterPairingInfo(destKubeconfigPath, token)
+	destInfo, err := volumeDriver.GetClusterPairingInfo(destKubeconfigPath, token, IsEks(), false)
```

Comment:
I don't see the changed GetClusterPairingInfo implementation in this PR

Reply:
This is from torpedo, here is the commit from torpedo:
https://github.com/portworx/torpedo/commit/f35cd96b7bae0393cdefa22daed32cdbffbc6c45